addon/index.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
We had agreed on this a week or so back, it's my fault for not recording it as an issue against the project